apps/blockchain/test/blockchain/state_test.exs

```diff
@@ -10,454 +10,445 @@ defmodule Blockchain.StateTest do
-      "stPreCompiledContracts/modexp_49_2401_2401_20500",
-      "stPreCompiledContracts/modexp_49_2401_2401_155000",
-      "stPreCompiledContracts/modexp_49_2401_2401_1000000",
-      "stPreCompiledContracts/modexp_3_5_100_35000",
-      "stPreCompiledContracts/modexp_3_5_100_25000",
-      "stPreCompiledContracts/modexp_3_5_100_22000",
-      "stPreCompiledContracts/modexp_3_5_100_20500",
-      "stPreCompiledContracts/modexp_3_5_100_155000",
-      "stPreCompiledContracts/modexp_3_5_100_1000000",
-      "stPreCompiledContracts/modexp_3_28948_11579_20500",
-      "stPreCompiledContracts/modexp_3_09984_39936_35000",
-      "stPreCompiledContracts/modexp_3_09984_39936_25000",
-      "stPreCompiledContracts/modexp_3_09984_39936_22000",
-      "stPreCompiledContracts/modexp_3_09984_39936_155000",
-      "stPreCompiledContracts/modexp_3_09984_39936_1000000",
-      "stPreCompiledContracts/modexp_39936_1_55201_35000",
-      "stPreCompiledContracts/modexp_39936_1_55201_25000",
-      "stPreCompiledContracts/modexp_39936_1_55201_22000",
-      "stPreCompiledContracts/modexp_39936_1_55201_20500",
-      "stPreCompiledContracts/modexp_39936_1_55201_155000",
-      "stPreCompiledContracts/modexp_39936_1_55201_1000000",
-      "stPreCompiledContracts/modexp_37120_37111_97_35000",
-      "stPreCompiledContracts/modexp_37120_37111_97_25000",
-      "stPreCompiledContracts/modexp_37120_37111_97_22000",
-      "stPreCompiledContracts/modexp_37120_37111_97_20500",
-      "stPreCompiledContracts/modexp_37120_37111_97_155000",
-      "stPreCompiledContracts/modexp_37120_37111_97_1000000",
-      "stPreCompiledContracts/modexp_37120_37111_37111_35000",
-      "stPreCompiledContracts/modexp_37120_37111_37111_25000",
-      "stPreCompiledContracts/modexp_37120_37111_37111_22000",
-      "stPreCompiledContracts/modexp_37120_37111_37111_20500",
-      "stPreCompiledContracts/modexp_37120_37111_37111_155000",
-      "stPreCompiledContracts/modexp_37120_37111_37111_1000000",
-      "stPreCompiledContracts/modexp_37120_37111_1_35000",
-      "stPreCompiledContracts/modexp_37120_37111_1_25000",
-      "stPreCompiledContracts/modexp_37120_37111_1_20500",
-      "stPreCompiledContracts/modexp_37120_37111_1_155000",
-      "stPreCompiledContracts/modexp_37120_37111_1_1000000",
-      "stPreCompiledContracts/modexp_37120_37111_0_35000",
-      "stPreCompiledContracts/modexp_37120_37111_0_25000",
-      "stPreCompiledContracts/modexp_37120_37111_0_22000",
-      "stPreCompiledContracts/modexp_37120_37111_0_20500",
-      "stPreCompiledContracts/modexp_37120_37111_0_155000",
-      "stPreCompiledContracts/modexp_37120_37111_0_1000000",
-      "stPreCompiledContracts/modexp_37120_22411_22000",
+      "stBugs/returndatacopyPythonBug_Tue_03_48_41-1432",
+      "stCreateTest/CreateOOGafterInitCodeReturndata",
       "stPreCompiledContracts/modexp",
-      "stCreateTest/CreateOOGafterInitCodeReturndata"
+      "stPreCompiledContracts/modexp_37120_22411_22000",
```

Comment:
Do you know what changed here? It's hard to see how many tests passed thanks to the fix you made.

Reply:
@germsvel around 40 tests are fixed related to `revert` operation